### PR TITLE
Remove user from all groups

### LINKED ISSUE #75: Add support to remove a user from All Groups
It would be great to be able to remove a user from all groups.

The REST API supports this: https://github.com/Azure/azure-signalr/blob/dev/docs/rest-api.md#remove-a-user-from-all-groups

Something could be added for the current SignalRGroupActions, such as SignalRGroupAction.RemoveFromAll.

### PR DESCRIPTION
**Pull request by brunhil**
Fixes #75 , by adding ability to remove a user from all groups.

### CODE REVIEW

File: src/SignalRServiceExtension/Bindings/SignalRGroupAction.cs [hunk 1]
```diff
@@ -33,6 +33,8 @@ public enum GroupAction
         [EnumMember(Value = "add")]
         Add,
         [EnumMember(Value = "remove")]
-        Remove
+        Remove,
+        [EnumMember(Value = "remove-all")]
```
Comment (realwanpengli) [inline src/SignalRServiceExtension/Bindings/SignalRGroupAction.cs]: how about `removeAll`

File: src/SignalRServiceExtension/Client/AzureSignalRClient.cs [hunk 2]
```diff
@@ -117,6 +117,16 @@ public async Task RemoveUserFromGroup(string userId, string groupName)
             await serviceHubContext.UserGroups.RemoveFromGroupAsync(userId, groupName);
         }
 
+        public async Task RemoveUserFromAllGroups(string userId)
```
Comment (realwanpengli) [inline src/SignalRServiceExtension/Client/AzureSignalRClient.cs]: How about `RemoveUserFromAllGroupsAsync`?

**Comment by realwanpengli** [comment]
Thank you for your contributing! Nice job!

**Comment by brunhil** [comment]
Any chance we can get this merged in? I'd love to use this!

**Comment by realwanpengli** [comment]
Merged

**Pull request merged**